platform/accessibility/common-issues/pdf/no-structural-tags.md

```diff
@@ -0,0 +1,101 @@
+
+## Tag your document
+
+Documents can be tagged in Microsoft Word as you create them. Use the built in tools to create headings, lists, and tables, then export your document as a tagged PDF.
+
+### Define headings in Microsoft Word
+
+Documents should have one **Title** and one **Heading 1**. **Heading 1** should be the main idea or message. Documents can have mutiple **Heading 2s** to communicate the most important supporting ideas. **Heading 3s** can be used for nested details.
+
+1. Select the **Home** tab
+2. Select or click and drag to highlight the text you want to convert to a heading
+3. Select the heading level you want
+4. Repeat steps 1-3 for additional headings
+5. Confirm no accessibility violations using the [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+
+![Clicking the Heading 1 button will convert a line of normal text to a heading level one](https://user-images.githubusercontent.com/934879/90903853-25f99500-e394-11ea-8146-161b90e31ff3.jpg)
+
+---
+
+### Define lists in Microsoft Word
+
+1. Select the **Home** tab
+2. Select or click and drag to highlight the text you want to convert to a list
+3. Select the Unordered List or Ordered List button to choose the type of list you want
+4. Repeat steps 1-3 for additional lists
+5. Confirm no accessibility violations using the [Microsoft Word accessibility panel](#microsoft-word-accessibility-panel)
+
+![Clicking the ordered list or unordered list button will convert normal text to a list](https://user-images.githubusercontent.com/934879/90903746-006c8b80-e394-11ea-84c7-2f45c1e1d6de.jpg)
+
+---
+
+### Export a tagged PDF
+
+1. Select the **Acrobat** tab
+2. Select **Create PDF**
+3. Name your document memorably
+4. Ensure the "Best for electronic distribution and accessibility" radio button is checked
+5. Select the **Export** button
+
+![Always select the radio button Best for electronic distribution and accessibility to export a tagged PDF](https://user-images.githubusercontent.com/934879/90903698-ea5ecb00-e393-11ea-8816-aa087b07602c.jpg)
+
+## Test for compliance
+
+### Microsoft Word accessibility panel
+
+We recommend you check your Word document for accessibility while working. Click the `Review` tab, then click the `Check Accessibility` button to open the Accessibility Checker panel.
+
+![Click the Review tab in Word, then click the Check Accessibility button](https://user-images.githubusercontent.com/934879/88831092-7a926180-d194-11ea-8b71-dce1300deb2d.jpg)
+
+---
+
+### Adobe Acrobat Pro accessibility tools
+
+Run the Acrobat Accessibility Checker by clicking the purple `Accessibility` button on the top right of your application window. Select **Accessibility Check** in the accessibility panel, ensure the **Document is tagged PDF** checkbox is checked, then select the **Start Checking** button. This will run the accessibility checker, and report any issues in a panel on the left of your document.
```

Comment:
This is how I would probably write it based on general best practices for this type of instructions, but it's not as descriptive as your explanation. If you think the additional info about the color and location of the buttons/tabs are necessary/helpful for users, feel free to adjust this format with more detail.

1. Go to **Accessibility > Accessibility Check**
1. Check the **Document is tagged PDF** checkbox
1. Select **Start Checking**  

This will run the accessibility checker, and report any issues in a panel on the left of your document.